BookingQueueSubscriber/BookingQueueSubscriber.Services/MessageHandlers/HearingReadyForVideoHandler.cs

```diff
@@ -35,12 +35,13 @@ public async Task HandleAsync(HearingIsReadyForVideoIntegrationEvent eventMessag
                 var newParticipantUsers = await _userCreationAndNotification.CreateUserAndNotifcationAsync(
                     eventMessage.Hearing, eventMessage.Participants);
                 
-                if (_featureToggles.UsePostMay2023Template() && newParticipantUsers is {Count: > 0})
+                if (_featureToggles.UsePostMay2023Template())
                 {
                     // The new template journey combines the account details and hearing details into one email.
                     // we need to remove the new user created in the previous step because they already received notifications for hearing
-                    await _userCreationAndNotification.SendHearingNotificationAsync(eventMessage.Hearing,
-                        eventMessage.Participants.Where(x => newParticipantUsers.All(y=>y.Username != x.Username)));
+                    var participants = eventMessage.Participants.Where(x =>
```

Comment:
if individuals are being filtered from this list, when do you send notifications for individuals?